_guides/general/useful-information/Clean-laptop.md

```diff
@@ -0,0 +1,29 @@
+---
+layout: wiki
+title: How to clean your Laptop
+description: "Specifying the methods to clean a laptop without damaging it"
+parent: Useful Information
+grand_parent: General
+image: https://media.discordapp.net/attachments/852210156875153408/955908527724068894/unknown.png
+contributors: ['M164'] 
+---
+
+# How to clean your Legion
+
+## Cleaning the fans
+
+1. Buy a can with compressed air made for cleaning electronics. This kind doesn't contain oil.
+2. Best to remove the back cover, so the dust can be properly pushed out of the laptop.
+3. When cleaning, prevent the fans from spinning. Can use a finger to very gently hold it in place. Holding them in place is to prevent the fans from spinning too fast, which could damage them.
+4. Don't hold the bottle upside to prevent liquid leaking form the can.
+5. Do it outside. The dust cloud that will get released from the laptop will be surreal. Even if it may seem like there is not much dust, the compressed air will push every last bit of it, from every laptop part in range, into the air.
+
+## CLeaning the keyboard deck
+
+1. A simple damp cloth should suffice. Just make sure the cloth is just damp and not soaked with water.
+2. Do not use any aggressive solution. Ideally no solution at all, to prevent damage to the surface and/or coating of the laptop chassis. Don't let liquid get into the laptop itself, so be extra careful when cleaning the keyboard, touchpad or around vents and exhaust.
+
+## Cleaning the Display Panel
+
+1. There are cleaning solutions with cleaning accessories made specifically for cleaning display panels. Easiest and best solution to buy those. They don't cost much and do the job well.
+2. For everyday cleaning, a microfiber cloth should suffice.
```

Comment:
```suggestion
## Cleaning the Display Panel

1. There are cleaning solutions with cleaning accessories made specifically for cleaning display panels. Easiest and best solution to buy those. They don't cost much and do the job well.
2. For everyday cleaning, a microfiber cloth should suffice.
3. When using the cloth, make sure it's actually clean. You don't want to use a dusty cloth, as the dust on it could scratch the display panel.
```